pybamm/expression_tree/concatenations.py

```diff
@@ -47,6 +49,19 @@ def get_children_domains(self, children):
 
         return domain
 
+    @property
+    def children(self):
```

Comment:
I'm unsure about overloading this. Presumably there are, or will be methods in Symbol which use `children` assuming that it gives the children *with* parents. Perhaps it can be called `children_without_parents`, or `orphans` even.

Reply:
ok, makes sense - `orphans` is a good extension of syntax! In that case, it makes sense to put this function in `Symbol` itself rather than just `Concatenation`?